recipes/open62541/all/conanfile.py

```diff
@@ -361,6 +377,12 @@ def package_info(self):
             os.path.join("include", "plugin")
         ]
 
+        #required for creating servers from ua nodesets
+        self.env_info.open62541_TOOLS_DIR = os.path.join(self.package_folder, "share", "tools")
```

Comment:
This is a cmake variable, not an environment variable.
https://github.com/open62541/open62541/search?q=open62541_TOOLS_DIR
I think you need to add this variable to a cmake script.
Like `lib/cmake/open62541/open62541Macros.cmake`, but then specific for this variable.

Reply:
Using an environment variable is problematic when using a 2 profile build (one for host and one for build machine), because environment variables from host requirements are not applied.

Reply:
Could you please elaborate how should I do this?